code/_hooks/events.dm

```diff
@@ -34,6 +34,8 @@
 	return handlers.Remove(key)
 
 /event/proc/Invoke(var/list/args)
+	if(handlers == null)
```

Comment:
Find what instances would cause handlers to be null.

Reply:
I'll save you the trouble on this one:
Go to the holodeck computer, set it to anything with holomobs in it, then to something else so that the old mobs are deleted.
This will start the runtimes.
The reason I didn't fix this is because I didn't find a solution that wasn't just adding `on_moved` to the list here:
https://github.com/vgstation-coders/vgstation13/blob/5bc40417d7d17642dd8a6d7a666925707dec0247/code/__HELPERS/unsorted.dm#L1004
Because that seemed like a band-aid at the time.